CHANGELOG.md

```
@@ -11,6 +11,75 @@ Version numbers follow [Semantic Versioning](https://semver.org/spec/v2.0.0.html
 
 ---
 
+## [0.16.0] — 2026-08-25
+
+### Added
+
+- `subdomains-top1million-full.txt` (4,730,673 labels) is now available in the
+  image at `/opt/SecLists/Discovery/DNS/`. SecLists ships it only as a `.7z`,
+  and its payload is a two-column frequency table (`   5617 www`) rather than a
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)